src/Ethereum.sol

```diff
@@ -85,7 +86,18 @@ library Ethereum {
     address internal constant WSTETH_DEBT_TOKEN        = 0xd5c3E3B566a42A6110513Ac7670C1a86D76E13E6;
 
     /******************************************************************************************************************/
-    /*** Implementation Addresses                                                                                   ***/
+    /*** SparkLend - Auxiliary Protocol Addresses                                                                   ***/
```

Comment:
I would also add the kill switch oracle: `0x909A86f78e1cdEd68F9c2Fe2c9CD922c401abe82`

Reply:
Added